.docs_overrides/main.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+
+{% block outdated %}
+  You're not viewing the latest release documentation.
```

Comment:
How about following the Ceph docs pattern for latest?
```suggestion
  This document is for a development version of Rook.
```